Automatic Validation ended with:
> 2024-05-03T20:43:45.6652098Z ##[error]Manifest Validation Failed
 2024-05-03T20:43:50.7470601Z ##[section]Finishing: Validate Manifest

(Automated response - build 880.)

--- a/manifests/a/ArduinoSA/IDE/stable/2.3.2/ArduinoSA.IDE.stable.installer.yaml
+++ b/manifests/a/ArduinoSA/IDE/stable/2.3.2/ArduinoSA.IDE.stable.installer.yaml
@@ -28,7 +28,7 @@ FileExtensions:
 - ixx
 ProductCode: '{245C87AB-B263-46D2-92BA-61E0F067D27E}'
 AppsAndFeaturesEntries:
-- DisplayVersion: 2.2.1
+- DisplayVersion: 2.3.2
   UpgradeCode: '{315F6168-1C48-5F6A-953D-BD5ADC41FE08}'
 Installers:
 - Architecture: x64

